images/airflow/2.9.2/python/mwaa/celery/task_monitor.py

```diff
@@ -362,6 +445,65 @@ def _get_current_task_count(self):
                 current_task_count += 1
         return current_task_count
 
+    def process_next_signal(self):
+        """
+        This method is used to process any signals sent by MWAA. This method processes the first signal it finds
+        in the chronological order of the available unprocessed signals.
+        """
+        if not self.mwaa_signal_handling_enabled:
+            logger.info("Signal handling is not enabled for this worker.")
```

Comment:
Which log stream will this show up in?

Reply:
All logger.info statements target service side logs stream here.